src/util/api.js

```diff
@@ -43,6 +43,19 @@ export const apiGetWidgetInstances = () => {
 		})
 }
 
+export const apiGetPaginatedWidgetInstances = page_number => {
```

Comment:
This function appears to do the job, it might be worth it to see if the `apiGetWidgetInstances` function is being used by anything else in Materia's front end, and if not, just replace `apiGetWidgetInstances` with the new version (but keeping the `apiGetWidgetInstances` name). Having both a paginated and non-paginated version would be redundant.

Reply:
The API is not used anywhere else, so I'll make that update.